ortools/glop/initial_basis.cc

```diff
@@ -99,27 +99,27 @@ void InitialBasis::CompleteBixbyBasis(ColIndex num_cols,
 
```

Comment:
I understand the source code has changed since the first PR.
 Can you remove this file from the PR.